internal/driver/glfw/loop.go

```diff
@@ -65,13 +65,17 @@ func (d *gLDriver) runGL() {
 
 	settingsChange := make(chan fyne.Settings)
 	fyne.CurrentApp().Settings().AddChangeListener(settingsChange)
-	d.initGLFW()
+	initOnce.Do(d.initGLFW)
 
 	for {
 		select {
 		case <-d.done:
+			runMutex.Lock()
 			fps.Stop()
 			glfw.Terminate()
+			runFlag = false
+			initOnce = &sync.Once{}
```

Comment:
not creating a new Instance would result in not initializing the glfw lib again, obviously...